meson.build

```diff
@@ -122,8 +122,8 @@ endif
 gstd_run_state_dir = '-DGSTD_RUN_STATE_DIR="@0@"'.format(runstatedir)
 
 logstatedir = get_option('with-gstd-logstatedir') + '/'
-if logstatedir == '${prefix}/var/log/gstd/'
-   logstatedir = prefix + '/var/log/gstd/'
+if logstatedir == '/var/log/gstd/'
+   logstatedir = '/var/log/gstd/'
```

Comment:
why prefix is not longer needed?

Reply:
I suspect it was added here bu mistake. The folders have always been `/var/log/gstd` and `/var/log/run/` and when meson was added they were changed to `/usr/var/log/gstd` and `/usr/var/log/run/` when using the `/usr` prefix. Autotools still uses the original folders, only meson was creating the folders on a different location. The problem with this is that the post install script was modifying the default location, so it was a matter of adding the prefix to the debian scripts and autotools or removing the prefix here.